dcpy/connectors/socrata/publish.py

```diff
@@ -267,6 +267,29 @@ def patch_metadata(
             )
         )
 
+    def _fetch_socrata_revision_data(self):
```

Comment:
does this replace `fetch_open_revisions()` [here](https://github.com/NYCPlanning/data-engineering/blob/878a2ea7107f689baf83c258c9844777d88b6d4a/dcpy/connectors/socrata/publish.py#L185)?

Reply:
Good questions! Unfortunately not. Added some documentation to hopefully clear up the usage [here](https://github.com/NYCPlanning/data-engineering/pull/947/commits/dff27e4ab609ffcfe8785bcf33b3b022699b965f)